docs/cli/text-definition.md

```diff
@@ -0,0 +1,173 @@
+# Defining tests as text files
+
+One important aspect of testing your code is the ability of adapting fast to changes while don't breaking your application. So, it is important that if you change your application, you are able to update your tests and run them against your new implementation as soon as possible to have a nice development feedback loop.
+
+As Tracetest is mainly a visual tool, this might make it difficult to update tests in an auditable way and executing those changes only when we are sure the application has being deployed with our changes. With that in mind, we build a new way for you to define your tests: using a YAML test definition!
+
+## Motivation
+
+Imagine that you were assigned to a ticket to improve your application database usage. You notice that everytime a specific endpoint is called, your application executes `N+1` select statements on the database instead of only one statement. You probably already have a test in place to ensure the correct functionality of that endpoint: it inserts the necessary information into the database and then call that specific endpoint using our tool and ensure you get the expected results using the trace generated by your application. It works fine, but there is a problem. That test is managed by Tracetest on its server and I cannot change the test until my patch is deployed, otherwise, if anyone tries to run that test using a non-patched version of the application, the test would fail.
```

Comment:
```suggestion
Imagine that you were assigned a ticket to improve your application database usage. You notice that every time a specific endpoint is called, your application executes `N+1` select statements on the database instead of only one statement. You probably already have a test in place to ensure the correct functionality of that endpoint: it inserts the necessary information into the database, calls that specific endpoint using our tool and ensures you get the expected results using the trace generated by your application. It works fine, but there is a problem. That test is managed by Tracetest on its server and the test cannot be changed until the new patch is deployed. Otherwise, if the test is run using a non-patched version of the application, the test would fail.
```